clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -627,7 +627,7 @@
   },
   "dependencies": {
     "@code4z/cobol-dialect-api": "file:../cobol-dialect-api",
-    "@code4z/analysis": "file:../analysis",
+    "@code4z/analysis": "file:../analysis/lib",
```

Comment:
Instead of doing this, please add `main` field to the `package.json` of the `@code4z/analysis` package like in https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/development/clients/cobol-dialect-api/package.json#L5